src/D2L.CodeStyle.Analyzers/ApiUsage/DangerousMemberUsages/DangerousMemberUsagesAnalyzer.cs

```diff
@@ -12,8 +12,8 @@ namespace D2L.CodeStyle.Analyzers.ApiUsage.DangerousMemberUsages {
 	[DiagnosticAnalyzer( LanguageNames.CSharp )]
 	internal sealed class DangerousMemberUsagesAnalyzer : DiagnosticAnalyzer {
 
-		private const string AuditedAttributeFullName = "D2L.CodeStyle.Annotations.DangerousMethodUsage+AuditedAttribute";
-		private const string UnauditedAttributeFullName = "D2L.CodeStyle.Annotations.DangerousMethodUsage+UnauditedAttribute";
+		private const string DangerousMethodAuditedAttributeFullName = "D2L.CodeStyle.Annotations.DangerousMethodUsage+AuditedAttribute";

```

Comment:
personally, I think this rename is overkill.  The string is `private`, and we're in a class named Dangerous*, so adding that on is redundant.  Unless you're going to be making this not private